sql/src/main/java/io/crate/protocols/postgres/Messages.java

```diff
@@ -200,6 +200,20 @@ static void sendErrorResponse(Channel channel, Throwable throwable) {
         final String message = Exceptions.messageOf(throwable);
         byte[] msg = message.getBytes(StandardCharsets.UTF_8);
         byte[] severity = "ERROR".getBytes(StandardCharsets.UTF_8);
+        byte[] lineNumber = null;
+        byte[] fileName = null;
+        byte[] methodName = null;
+
+        if (throwable.getStackTrace() != null) {
```

Comment:
Please assign the getStackTrace result to a local variable. Although it is creating the stacktrace only once it's still synchronized and cloning the thing.